pybikes/metro_bike_share.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+class MetroBikeShare(BikeShareSystem):
+
+    sync = True
+
+    meta = {
+        'system': 'Metro Bike Share',
+        'company': 'Bicycle Transit Systems, Inc.',
+    }
+
+    def __init__(self, tag, feed_url, meta):
+        super(MetroBikeShare, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            response = scraper.session.request(
+            # some endpoints might fail verification, so it's up to the spider
+            # to disable it
+            url=self.feed_url,
+            method='GET',
+            verify=False,
+        )
```

Comment:
This is not how this is supposed to work. We initialize an scraper in case we are not provided an scraper. Request happens on both cases